### ISSUE TITLE
Add a hint about the correct format of a link as cell input

### ISSUE THREAD

**Issue by silverkszlo** [nextcloud/tables#2001]
When a user wants to add a link to a table, the format constraint requires the string to start with "https://...". The input field will not accept entries like "google.com". If a user tries to save an incorrectly formatted link, the edit field simply closes without showing an error message and, obviously, without saving the invalid input.

<img width="1103" height="295" alt="Image" src="https://github.com/user-attachments/assets/c908ef3b-ab9a-41c9-abf1-58ffece9f26f" />

It would be helpful to provide a hint for the user, for example as a placeholder, to indicate the correct format. Additionally, if the user attempts to save a wrongly formatted input, an error message should be displayed.
